generators/entity/index.js

```diff
@@ -738,6 +738,9 @@ class EntityGenerator extends BaseBlueprintGenerator {
                 if (!context.relationships) {
                     context.relationships = [];
                 }
+                var humanDatabaseNames = {"mongodb": 'MongoDB', 'cassandra': 'Cassandra', 'sql': 'SQL', 'couchbase': 'Couchbase', 'neo4j': 'Neo4j'};
```

Comment:
This should be factored with `server/prompts.js` and probably moved to `generator-constants.js`, but it should be a different PR. 
Meanwhile, please move it to `entity-server` since there's no use for `entity-client` so it doesn't need to be in `entity` subgen. And I suggest `officialDatabaseNames`, because `humanDatabaseNames` is a bit weird. Or `humanReadableDatabaseNames` but that's longer.